templates/default/capture-agent-manager-gunicorn-start.sh.erb

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+NAME="<%= @capture_agent_manager_name %>"
+USER="<%= @capture_agent_manager_usr_name %>"
+GROUP=${USER}
+FLASKDIR="/home/${USER}/sites/${NAME}"
+VENVDIR="${FLASKDIR}/venv"
+SOCKFILE="/home/${USER}/sock/${NAME}.sock"
+NUM_WORKERS=3
+
+echo "starting ${NAME}"
+
+# activate virtualenv
+source ${VENVDIR}/bin/activate
+
+export PYTHONPATH=${FLASKIDR}:${PYTHONPATH}
+
+# export all env vars for app
+source ${FLASKDIR}/${NAME}.env
```

Comment:
i think it used to be just variable=foo when it was set up for heroku... what should be extension then? `.sh`? i guess i'm still learning these patterns to setup stuff in the env.